config/config.txt

```diff
@@ -1,35 +1,35 @@
-## Server name: This appears at the top of the screen in-game and in the BYOND hub. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice.
+## Server name: This appears at the top of the screen in-game and in the BYOND hub. Activate and replace 'tgstation' with the name of your choice.
```

Comment:
Uh no, these comments are for people not using your tool, please leave them alone

Your tool readme should instruct people to not touch these config lines :-1: